src/test/java/de/mediathekview/mserver/crawler/orf/tasks/OrfFilmDetailTaskTestWithEpisodes.java

```diff
@@ -0,0 +1,215 @@
+package de.mediathekview.mserver.crawler.orf.tasks;
+
+import de.mediathekview.mlib.daten.Film;
+import de.mediathekview.mlib.daten.GeoLocations;
+import de.mediathekview.mlib.daten.Sender;
+import de.mediathekview.mserver.testhelper.AssertFilm;
+import de.mediathekview.mserver.testhelper.JsoupMock;
+import java.io.IOException;
+import java.time.Duration;
+import java.time.LocalDateTime;
+import java.time.temporal.ChronoUnit;
+import java.util.Set;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import org.jsoup.Jsoup;
+import static org.junit.Assert.assertThat;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({Jsoup.class})
+@PowerMockIgnore("javax.net.ssl.*")
+public class OrfFilmDetailTaskTestWithEpisodes extends OrfFilmDetailTaskTestBase {
+  
+  private static final String REQUEST_URL = "http://tvthek.orf.at/profile/Aktuell-in-Oesterreich/13887571/Aktuell-in-Oesterreich/13962830";
+  private static final String EXPECTED_THEME = "Aktuell in Österreich";
+  private static final LocalDateTime EXPECTED_TIME = LocalDateTime.of(2018, 1, 26, 17, 5, 0);
+  private static final GeoLocations[] EXPECTED_GEO = new GeoLocations[] { GeoLocations.GEO_NONE };
+
+  private static final int INDEX_TITLE = 0;
+  private static final int INDEX_DURATION = 1;
+  private static final int INDEX_DESCRIPTION = 2;
+  private static final int INDEX_SUBTITLE = 3;
+  private static final int INDEX_URL_SMALL = 4;
+  private static final int INDEX_URL_NORMAL = 5;
+  private static final int INDEX_URL_HD = 6;
+  
+  private static Object[][] data() {
+    return new Object[][] { 
+      {
+        "Aktuell in Österreich",
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Use already-defined constant 'EXPECTED_THEME' instead of duplicating its value here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)